frontend/components/Logo/logo.module.css

```diff
@@ -0,0 +1,4 @@
+.logo {
+  margin: 0 auto;
+  width: 160px;
```

Comment:
I think we're using `rem` over `px` - but my last merge introduced some variables you can use - `var(--size160)` will map to the right `rem` value for 160px!

Reply:
Yeah, I was actually waiting for your branch to get merged so I could fold in any styling you added that overlapped, and also see what conventions you were using to hew closer to those.